checkstyle/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "src"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+regex = "1.9.1"
+futures = "0.3"
+tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
+text_io = "0.1.12"
+clap = { version = "4.3.22", features = ["derive"] }
```

Comment:
Let's make sure our files end with new lines. Funnily enough, this tool will eventually help prevent this from happening.